plugins/amdgpu/amdgpu_plugin.c

```diff
@@ -810,10 +815,104 @@ static int dump_process(int fd, struct kfd_ioctl_criu_process_info_args *info_ar
 	return ret;
 }
 
+static int dump_devices(int fd,
+			struct kfd_ioctl_criu_process_info_args *info_args,
+			CriuKfd *e)
+{
+	int ret = 0, i, j;
+	struct kfd_ioctl_criu_dumper_args args = {0};
```

Comment:
reverse x-max everywhere, though its not a rule but a convention that I prefer and many in kernel community also recommend.

Reply:
Most of this file was not following this convention.
Changed it for the new functions that I added